cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -100,6 +100,12 @@ spec:
           - name: "ESS_TLS_SERVER_SECRET_NAME"
             value: ess-server-certs
           {{- end }}
+          - name: "TLS_CA_SECRET_NAME"
+            value: root-ca-certs
```

Comment:
Should this secret have "crossplane" in its name?

Reply:
Updated to `xp-root-ca`. I can also make it `crossplane-root-ca` if you prefer.

Reply:
+1 for crossplane-root-ca